pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/AdditionTransformFunction.java

```diff
@@ -74,6 +74,7 @@ public void init(List<TransformFunction> arguments, Map<String, ColumnContext> c
     if (_resultDataType == DataType.BIG_DECIMAL) {
       _literalBigDecimalSum = _literalBigDecimalSum.add(BigDecimal.valueOf(_literalDoubleSum));
     }
+    super.init(arguments);
```

Comment:
(convention) We usually call `super.init()` as first statement

Reply:
done